.github/workflows/release.yml

```
@@ -8,30 +8,31 @@ on:
         description: Bump version number
         options: [major, minor, patch]
         default: patch
-  pull_request:
-    types: [closed]
+  pull_request: { types: [closed] }
 jobs:
   bump:
     if: github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - run: |
-          git config --global user.name Olivaw[bot]
-          git config --global user.email 64868532+iterative-olivaw@users.noreply.github.com
+      - uses: actions/checkout@v3
+      - uses: iterative/setup-cml@v1
+      - name: Bump version
+        run: |
+          cml ci
           git checkout -b bump/$(npm version ${{ github.event.inputs.bump }})
+      - name: Create PR
+        run: |
           git push --set-upstream origin --follow-tags HEAD
-          gh pr create --title "Bump version to $(git describe --tags)" --body "Approve me 🤖"
-          gh pr merge --squash --auto
+          cml pr create --title "Bump version to $(git describe --tags)" --body "Approve me 🤖" --squash
         env:
           GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
   release:
     if: github.event.pull_request.merged && startsWith(github.head_ref, 'bump/')
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - run:
-          gh release create --generate-notes {--title=CML\ ,}$(basename ${{
-          github.head_ref }})
+          gh release create --draft --generate-notes {--title=CML\ ,}$(basename
+          ${{ github.head_ref }})
```

do we need to wrap this line?

```suggestion
          gh release create --draft --generate-notes {--title=CML\ ,}$(basename ${{ github.head_ref }})
```

---

Probably not. It's a consequence of Codespaces' editor settings. 🙈

---

the pre-commit hooks/linter insists on this wrapping. Would be happy if anyone wants to mod `.eslintrc.js`/`prettier.config.js/`.restyled.yaml`...

---

btw also if you run `npm run lintfix` locally there are loads of errors... worth a follow-up issue?

---

Both of your last comments are worth a follow-up issue. I'm not a fan of that wrapping either.

---

done: #1177, #1178